### DISCUSSION TITLE
monaco binary file

### DISCUSSION THREAD

**Discussion by bita-94** [Dynatrace/dynatrace-configuration-as-code#322] [Archive] [answered]
Hi,
I use a mac machine and try to install Monaco, but after downloading that, Monaco binary was not available. Could you please help me. 
Thanks.

**Comment by patrickpichler** [comment, marked as answer]
Hi, 
What version of Monaco did you download?

**Reply by bita-94** [reply]
Hi 
I download Release v1.5.3 for both windows and mac

**Reply by mdicss** [reply]
Monaco is only one executable file. If you wish to have a command called 'monaco', you probabely have to rename the downloaded file. Maybe you have to change permissions too.
